docs/documentation/development/bindings/xml-reference.md

```diff
@@ -101,6 +101,16 @@ The following HTML tags are allowed -: ```<b>, <br>, <em>, <h1>, <h2>, <h3>, <h4
   <tr><td>criteria.name</td><td>The name of the context related filter.</td></tr>  
 </table>
 
```

Comment:
Maybe add a (small) header here and link to this section from the context entry in the table above?